topcoffea/modules/event_selection.py

```diff
@@ -46,7 +48,10 @@ def trg_pass_no_overlap(events,is_data,dataset,year,dataset_dict,exclude_dict):
     trg_passes = passes_trg_inlst(events,full_trg_lst)
 
     # In case of data, check if events overlap with other datasets
-    if is_data:
+    if is_data and (era is not None):   #Used for potential era dependency in Run3
+        trg_passes = passes_trg_inlst(events,dataset_dict[year][dataset])
```

Comment:
If you are checking by era in this case, should the passes_trg_inlist also check era here? (Like you do for overlap check?)

Reply:
No, because the total trigger list will be the same for each era